lib/actions/local/push.coffee

```diff
@@ -43,12 +43,14 @@ resinPushHelp = '''
 	Here is an example '.resin-sync.yml' :
 
 		$ cat $PWD/.resin-sync.yml
-		destination: '/usr/src/app'
-		before: 'echo Hello'
-		after: 'echo Done'
-		ignore:
-			- .git
-			- node_modules/
+		local_resinos:
+			app-name: local-app
+			build-triggers:
+				- Dockerfile: abcdefabcdefabcdefabcdefabcdefabcdefabcdef
+				- package.json: abcdefabcdefabcdefabcdefabcdefabcdefabcdef
```

Comment:
Could these `abcdefabcdefabcdefabcdefabcdefabcdefabcdef` be a bit more descriptive on what the expected value should be?